app/utils/get_filtered_records.py

```diff
@@ -190,7 +190,15 @@ def date_filter(record, start_date=None, end_date=None, use_sampling_date=True):
     if prioritize_estimates:
         result_df = pd.DataFrame(result)
         prioritized_records = get_prioritized_estimates(result_df, mode=prioritize_estimates_mode)
+        # If records exist, clean dataframe
         if not prioritized_records.empty:
+            # Convert from True/None to True/False
+            # TODO: maybe this is something we enforce in the ETL moving fwd
+            bool_fields = ['academic_primary_estimate', 'dashboard_primary_estimate', 'test_adj', 'pop_adj', 'geo_exact_match']
```

Comment:
Is it worth it to generate bool fields automatically, so this doesn't break later if we add another bool field?

Could iterate over the columns of prioritized_records to see which have boolean dtypes, and apply this transformation to them

Reply:
yeah i think this is better